Doc/library/curses.rst

```diff
@@ -881,14 +884,15 @@ the following methods and attributes:
 
    Get a character. Note that the integer returned does *not* have to be in ASCII
    range: function keys, keypad keys and so on return numbers higher than 256. In
-   no-delay mode, -1 is returned if there is no input, else :func:`getch` waits
+   no-delay mode, ``-1`` is returned if there is no input, else the function waits
```

Comment:
We should fix the tense here. The first part of the paragraph is in imperative mode, so this should say `In no-delay mode, return ``-1`` if there is no input, otherwise wait until a key is pressed`.  I might also change "return numbers higher than 256" to "are represented by numbers higher than 265".